course02/homework01/gavin/list.cpp

```diff
@@ -0,0 +1,143 @@
+#include <memory>
+
+#include <memory>
```

Comment:
This should be included once, and after your list.h. Includes should always be done from most to least specific.